plugin/build.gradle

```diff
@@ -37,7 +37,7 @@ sourceSets {
 }
 
 dependencies {
-    compile 'org.codehaus.groovy:groovy-all:2.4.10'
+    compile localGroovy()
     compile "org.jenkins-ci.plugins:job-dsl-core:1.61"
```

Comment:
Unrelated, but mind adding a new commit that uses single quotes here, while we're touching the code location?